corehq/apps/export/system_properties.py

```diff
@@ -216,6 +216,19 @@
         help_text=_('Link to this form'),
         selected=True,
     ),
+    ExportColumn(
+        tags=[PROPERTY_TAG_INFO],
+        label="web_user",
+        item=ExportItem(
+            path=[
+                PathNode(name='auth_context'),
+                PathNode(name='user_id'),
+            ],
+            transform=USERNAME_TRANSFORM,
+        ),
+        help_text=_('The web user who submitted the form'),
```

Comment:
From our slack convo:
Since this can be a mobile user, it might be clearer to reword this help text.

Also, given how confused everyone already is about web users vs mobile, can we possibly rename the column to something like hq_user, or something else that implies “the username they used to log in to our website”